mix.exs

```diff
@@ -55,7 +55,8 @@ defmodule EpochElixir.Mixfile do
 
       # aecore deps, they are needed here, as we can only override umbrella-apps dependencies here (https://github.com/elixir-lang/elixir/issues/5232)
       {:idna, github: "aeternity/erlang-idna", tag: "5.1.1", manager: :rebar, override: true},
-      {:ranch, github: "ninenines/ranch", tag: "1.4.0", override: true}
+      {:ranch, github: "ninenines/ranch", tag: "1.4.0", override: true},
+      {:ex_parameterized, "~> 1.3.2"}
```

Comment:
what is this used for?

Reply:
for the Vm tests; the tests were commented out by mistake in another PR, after that this dependency was removed with the unused dependencies PR; now I brought them both back

Reply:
:+1: